revenue_app/views.py

```diff
@@ -1,3 +1,4 @@
+import json
```

Comment:
I think this is not in use, you can safely remove it

Reply:
It's used in the last two function based views

Reply:
nvm then, completely missed that one